### ISSUE TITLE
Table Visibility

### ISSUE THREAD

**Issue by arpitchaudhari** [remirror/remirror#785]
### Summary
Not able to see table in editor.

### Steps to reproduce
Insert table in editor

### Expected results
Table should be visible so that user can enter data.

### Actual results
Not able to see table in editor. CSS is properly applied but table is not shown in editor.

### Screenshot(s)
![Screenshot 2020-11-18 at 8 46 14 AM](https://user-images.githubusercontent.com/29978858/99478746-f5b59880-297a-11eb-81a5-7de703e19bcf.png)

**Comment by ifiokjr** [comment]
I'd be open to a PR to fix this if you have time.

**Comment by arpitchaudhari** [comment]
Remirror css is not applied. That's why its not working. 

I have followed this doc [https://remirror.io/docs/guide/installation#required-styles](url) but its not working.
![Screenshot 2020-11-19 at 7 59 50 AM](https://user-images.githubusercontent.com/29978858/99613274-41307b00-2a3d-11eb-95d2-9b009d5d4a32.png)

To test it I have applied global css and table is visible.
![Screenshot 2020-11-19 at 8 02 59 AM](https://user-images.githubusercontent.com/29978858/99613515-b4d28800-2a3d-11eb-9dda-b5dde3b87aba.png)
![Screenshot 2020-11-19 at 8 03 47 AM](https://user-images.githubusercontent.com/29978858/99613561-c3b93a80-2a3d-11eb-801e-b7ba9f2025b4.png)

So I believe that remirror css is not imported properly.  I am finding reason why its not importing remirror css.
